Src/static/scripts.js

```
@@ -3,202 +3,218 @@ const FEED_FILTER_KEY = "feedFilter";
+  }
 
-    set limitValue(value) {
-        const limit = parseInt(value, 10);
-        if (isNaN(limit) || limit < 1) {
-            throw new Error("Invalid workflow limit value. Must be a number greater than 0.");
-        }
-        this._limitValue = limit;
-        this.saveLimiterValue(limit);
+  set limitValue(value) {
+    const limit = parseInt(value, 10);
+    if (isNaN(limit) || limit < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)